staff/gustavo-rojas/course-3/task-3/site-nysl/src/components/gameinfo.js

```diff
@@ -0,0 +1,183 @@
+      
+   },
+      {
+         "id":"04",  
+         "label": "Phone:",
+         "info": "(630) 690-8132",
+   },
+   {
+      "id":"05",
+      "label": "Email:",
+      "info": "michael.randall@chisoccer.org"
+   }
+   , {
+      "id":"06",
+      "label": "",
+      "info": ""
+   }
+   
+]
+
+export const game_info = [
+   {
+      "id": "001",
+      "Date": "2018-09-01",
+      "Teams": "U1 and U4",
+      "Location": "AJ Katzenmaier",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "002",
+      "Date": "2018-09-01",
+      "Teams": "U3 and U2",
+      "Location": "Greenbay",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "003",
+      "Date": "2018-09-08",
+      "Teams": "U5 and U6",
+      "Location": "Howard A Yeager",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "004",
+      "Date": "2018-09-08",
+      "Teams": "U6 and U1",
+      "Location": "Marjorie P Hart",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "005",
+      "Date": "2018-09-15",
+      "Teams": "U2 and U4",
+      "Location": "North",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "006",
+      "Date": "2018-09-15",
+      "Teams": "U3 and U5",
+      "Location": "AJ Katzenmaier",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "007",
+      "Date": "2018-09-22",
+      "Teams": "U1 and U3",
+      "Location": "South",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "008",
+      "Date": "2018-09-22",
+      "Teams": "U2 and U6",
+      "Location": "Howard A Yeager",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "009",
+      "Date": "2018-09-29",
+      "Teams": "U4 and U5",
+      "Location": "Greenbay",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "010",
+      "Date": "2018-10-06",
+      "Teams": "U2 and U5",
+      "Location": "Marjorie P Hart",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "011",
+      "Date": "2018-10-06",
+      "Teams": "U1 and U6",
+      "Location": "South",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "012",
+      "Date": "2018-10-13",
+      "Teams": "U3 and U4",
+      "Location": "Howard A Yeager",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "013",
+      "Date": "2018-10-13",
+      "Teams": "U5 and U1",
+      "Location": "Greenbay",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "014",
+      "Date": "2018-10-20",
+      "Teams": "U6 and U3",
+      "Location": "North",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "015",
+      "Date": "2018-10-20",
+      "Teams": "U2 and U4",
+      "Location": "Marjorie P Hart",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "016",
+      "Date": "2018-10-27",
+      "Teams": "U3 and U1",
+      "Location": "AJ Katzenmaier",
+      "Times": "9:30 a.m."
+   },
+   {
+      "id": "017",
+      "Date": "2018-10-27",
+      "Teams": "U5 and U6",
+      "Location": "Howard A Yeager",
+      "Times": "1:00 p.m."
+   },
+   {
+      "id": "018",

```

Comment:
As we agreed, this empty code will be removed on task 4.

Reply:
This is change is done . And added the bootstrap class to help to show properly the content of the page .